docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,142 @@
 ---
 description: >
-  Contains the definitions shared across design documents around prototyping a kube-like control plane (in KCP).  This is
-  a derivative work of other design documents intended to frame terminology.  All future statements that may be changed by
-  designs is covered by those designs, and not duplicated here.
+  The Terminology document contains definitions used throughout the kcp project. We strongly recommend familiarizing
+  yourself with this terminology before proceeding. Note that some terms might get changed in the future, and that
+  will be covered by respective design documents.
 ---
 
 # Terminology
 
+This document contains definitions used throughout the kcp project. We strongly recommend familiarizing yourself with
+this terminology before proceeding. Note that some terms might get changed in the future, and that will be covered by
+respective design documents.
+
 ## Logical Cluster
 
-A logical cluster is a way to subdivide a single kube-apiserver + etcd storage into multiple clusters (different APIs,
-separate semantics for access, policy, and control) without requiring multiple instances.  A logical cluster is a
-mechanism for achieving separation, but may be modelled differently in different use cases.  A logical cluster is
-similar to a virtual cluster as defined by sig-multicluster, but is able to amortize the cost of a new cluster to be
-zero or near-zero memory and storage so that we can create tens of millions of empty clusters cheaply.
+A logical cluster is a way to subdivide a single `kube-apiserver` instance and `etcd` datastore into multiple
+clusters (different APIs, separate semantics for access, policy, and control) without requiring multiple instances.
```

Comment:
nit: this was a little misleading before already: technically it is not kube-apiserver, but a kube-like apiserver.